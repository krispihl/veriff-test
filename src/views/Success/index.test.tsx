import React from 'react';
import '@testing-library/jest-dom';
import { render, screen } from '@testing-library/react';
import { SuccessView } from './index';

describe('Success', () => {
    it('renders Success view correctly', () => {
        const { container } = render(<SuccessView />);

        expect(container).toMatchSnapshot();
        expect(screen.getByText('Great success!', { exact: true})).toBeInTheDocument();
        expect(screen.getByText('Your data was submitted successfully.', { exact: true})).toBeInTheDocument();
    });
})