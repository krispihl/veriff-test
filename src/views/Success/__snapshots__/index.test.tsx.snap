// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Success renders Success view correctly 1`] = `
<div>
  <div
    class="container"
  >
    <h2>
      Great success!
    </h2>
    <p>
      Your data was submitted successfully.
    </p>
  </div>
</div>
`;
